src/site/layout/css.rs

```diff
@@ -24,7 +25,8 @@ fn concat_minify(css_files: &[String]) -> Result<String> {
     Ok(css)
 }
 
-pub fn build_oranda(dist_dir: &str) -> Result<Box<link<String>>> {
+pub fn build_oranda(config: &Config) -> Result<Box<link<String>>> {
```

Comment:
i might just pass dist_dir and path_prefix here so it's clear what this function needs